app/controllers/categories_controller.rb

```diff
@@ -0,0 +1,36 @@
+class CategoriesController < ApplicationController
+before_action :require_login, only: [:new, :create, :edit, :update]
+before_action :navbar_categories, only: [:index, :show]
+
+  def index
+    @cat = Category.all
+  end
+  def show
```

Comment:
Leave a space between methods and be consistent across the file.  It makes it easier to read.